lib/editors/editor_detector.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
  * @param root {Object} the document's root.
  * @return {string} the editor's name.
  */
-function detect(root) {
+async function detect(root) {
```

Comment:
perhaps it would be better to refactor this to remove the path where we ask the user to choose the editor.

Reply:
I will do it

Reply:
I get back to this, I don't see what you want. I think we must  split in different Merge request